docs/source/usage.rst

```diff
@@ -33,6 +33,32 @@ For example:
 
    print(prediction_uuid, ml_output) # Print the returned output
 
+Listing Past Predictions
+-------------------------
+
+To list past predictions, make use of the ``konan_sdk.sdk.KonanSDK.get_predictions()`` method, which
+uses `yield`-based semantics to both allow you to retrieve a large number of predictions while
+also maintaining low request latency and response size.
+
+.. code-block:: python
+
+   import datetime
+   from konan_sdk.sdk import KonanSDK
+
+   sdk = KonanSDK(verbose=False) # Initialize the SDK.
+   user = sdk.login("<email>", "<password>") # Login user your valid konan credentials
+
+   predictions_generator = sdk.get_predictions(
+      deployment_uuid="<deployment-uuid>",
+      start_time=datetime.datetime(year=2022, month=9, day=1),
+      end_time=datetime.datetime(year=2022, month=10, day=1),
+   )
+
+   for predictions in predictions_generator:
+      print(len(predictions))
+      print(predictions[0].uuid, predictions[0].features)
```

Comment:
An average user might be confused with the generator syntax, maybe rename `predictions` to `predictions_page` for clarity or add a small comment explaining that the generator iterates over pages and each page you can iterate over the predictions themselves.

Reply:
Done